services/app-api/serverless.yml

```diff
@@ -227,7 +230,8 @@ functions:
           method: post
           cors: true
           authorizer: aws_iam
-    timeout: 30 # zipping large amount of files hits default time out of 6 seconds
+    timeout: 30
```

Comment:
is 30s the longest we can get?

Reply:
It's an API gateway limit, not a Lambda one. Since it's attached to the API gateway we're stuck with a max of 30. Lambda itself I think is 15 minutes.